docs/configuration/config-file-reference.md

```diff
@@ -4353,6 +4353,15 @@ The `redis_config` configures the Redis backend cache. The supported CLI flags `
 # CLI flag: -<prefix>.redis.master-name
 [master_name: <string> | default = ""]
 
+# SRV service used to discover redis servers.
+# CLI flag: -<prefix>.redis.service
+[service: <string> | default = "redis"]
+
+# EXPERIMENTAL: Comma separated addresses list in DNS Service Discovery format:
```

Comment:
Does it need to be experimental?

Reply:
I think I copy/pasted it from the memcached DNS discovery docs.
I'll remove the experimental part.